README.md

```diff
@@ -1089,59 +1089,59 @@ Listing of the main packages used in the IPFS ecosystem. There are also three sp
-| [`ipfs-mfs`](//github.com/ipfs/js-ipfs-mfs) | [![npm](https://img.shields.io/npm/v/ipfs-mfs.svg?maxAge=86400&style=flat-square)](//github.com/ipfs/js-ipfs-mfs/releases) | [![Deps](https://david-dm.org/ipfs/js-ipfs-mfs.svg?style=flat-square)](https://david-dm.org/ipfs/js-ipfs-mfs) | [![Travis CI](https://travis-ci.com/ipfs/js-ipfs-mfs.svg?branch=master)](https://travis-ci.com/ipfs/js-ipfs-mfs) | [![codecov](https://codecov.io/gh/ipfs/js-ipfs-mfs/branch/master/graph/badge.svg)](https://codecov.io/gh/ipfs/js-ipfs-mfs) | [Alex Potsides](mailto:alex.potsides@protocol.ai) |
-| [`ipfs-unixfs`](//github.com/ipfs/js-ipfs-unixfs) | [![npm](https://img.shields.io/npm/v/ipfs-unixfs.svg?maxAge=86400&style=flat-square)](//github.com/ipfs/js-ipfs-unixfs/releases) | [![Deps](https://david-dm.org/ipfs/js-ipfs-unixfs.svg?style=flat-square)](https://david-dm.org/ipfs/js-ipfs-unixfs) | [![Travis CI](https://travis-ci.com/ipfs/js-ipfs-unixfs.svg?branch=master)](https://travis-ci.com/ipfs/js-ipfs-unixfs) | [![codecov](https://codecov.io/gh/ipfs/js-ipfs-unixfs/branch/master/graph/badge.svg)](https://codecov.io/gh/ipfs/js-ipfs-unixfs) | [Alex Potsides](mailto:alex.potsides@protocol.ai) |
+| [`ipfs-unixfs-exporter`](//github.com/ipfs/js-ipfs-unixfs-exporter) | [![npm](https://img.shields.io/npm/v/ipfs-unixfs-exporter.svg?maxAge=86400&style=flat-square)](//github.com/ipfs/js-ipfs-unixfs-exporter/releases) | [![Deps](https://david-dm.org/ipfs/js-ipfs-unixfs-exporter.svg?style=flat-square)](https://david-dm.org/ipfs/js-ipfs-unixfs-exporter) | [![Travis CI](https://flat.badgen.net/travis/ipfs/js-ipfs-unixfs-exporter.svg?branch=master)](https://travis-ci.com/ipfs/js-ipfs-unixfs-exporter) | [![codecov](https://codecov.io/gh/ipfs/js-ipfs-unixfs-exporter/branch/master/graph/badge.svg?style=flat-square)](https://codecov.io/gh/ipfs/js-ipfs-unixfs-exporter) | [Alex Potsides](mailto:alex.potsides@protocol.ai) |
+| [`ipfs-unixfs-importer`](//github.com/ipfs/js-ipfs-unixfs-importer) | [![npm](https://img.shields.io/npm/v/ipfs-unixfs-importer.svg?maxAge=86400&style=flat-square)](//github.com/ipfs/js-ipfs-unixfs-importer/releases) | [![Deps](https://david-dm.org/ipfs/js-ipfs-unixfs-importer.svg?style=flat-square)](https://david-dm.org/ipfs/js-ipfs-unixfs-importer) | [![Travis CI](https://flat.badgen.net/travis/ipfs/js-ipfs-unixfs-importer.svg?branch=master)](https://travis-ci.com/ipfs/js-ipfs-unixfs-importer) | [![codecov](https://codecov.io/gh/ipfs/js-ipfs-unixfs-importer/branch/master/graph/badge.svg?style=flat-square)](https://codecov.io/gh/ipfs/js-ipfs-unixfs-importer) | [Alex Potsides](mailto:alex.potsides@protocol.ai) |
```

Comment:
As part of this update, wanna move the 2 lines that js-ipfs-unixfs-engine does to js-ipfs itself? https://github.com/ipfs/js-ipfs-unixfs-engine/issues/237#issuecomment-552729429

Reply:
Not sure I understand? It doesn’t use the engine (and hasn’t for a long while) and already requires them separately.

Reply:
Gotcha! https://github.com/ipfs/js-ipfs/blob/master/package.json#L108-L110

Then https://github.com/ipfs/js-ipfs-unixfs-engine should be deprecated, no?

Reply:
👍 I'll open an issue. As I mentioned in the PR description it's out of date (depends on old versions of importer/exporter) so I'm sure this will be fine.

Reply:
https://github.com/ipfs/js-ipfs-unixfs-engine/issues/238